packages/thread-context/worker.js

```
@@ -0,0 +1,81 @@
+/**
+ * @module thread-context/worker
+ * @summary Worker-thread entry point. Reconstructs the Meteor API surface
+ * from a transferred MessagePort.
+ */
+
+import { BridgeClient } from './bridge-client.js';
+import { createCollectionProxy } from './proxies/collection-proxy.js';
+import { createMethodProxy } from './proxies/method-proxy.js';
+import { MeteorError } from './errors.js';
```

ESLint

[error] `./errors.js` import should occur before import of `./proxies/collection-proxy.js`

(import/order)